### ISSUE TITLE
Confusing output when one rule in command-line option list is mis-spelled

### ISSUE THREAD

**Issue by magnumripper** [magnumripper/JohnTheRipper#3935]
```
$ ../run/john -stdout -w -rules:wordlist,signle
Using default input encoding: UTF-8
No "wordlist,signle" mode rules found in ../run/john.conf
```
It's not obvious from the output that "wordlist" does exist but "signle" doesn't.

**Comment by magnumripper** [comment]
I might take a quick look at this when I can, for possibly fixing before release.

**Comment by magnumripper** [comment]
After KISS fix in #3936:
```
$ ../run/john -stdout -w -rules:wordlist,signle
Using default input encoding: UTF-8
"signle" not found; No "wordlist,signle" mode rules found in ../run/john.conf
```
